slasher/db/kv/spanner_test.go

```diff
@@ -0,0 +1,296 @@
+package kv
+
+import (
+	"context"
+	"flag"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/prysmaticlabs/prysm/slasher/detection/attestations/types"
+	"github.com/prysmaticlabs/prysm/slasher/flags"
+	"github.com/urfave/cli"
+)
+
+type spanMapTestStruct struct {
+	epoch   uint64
+	spanMap map[uint64]types.Span
+}
+
+var spanTests []spanMapTestStruct
+
+func init() {
+	spanTests = []spanMapTestStruct{
+		{
+			epoch: 1,
+			spanMap: map[uint64]types.Span{
+				1: {MinSpan: 10, MaxSpan: 20, SigBytes: [2]byte{0, 1}},
```

Comment:
HasAttested should be true for this case

Reply:
it is just to check that the db stores what it is being given